Assets/SEE/Game/Worlds/PlayerSpawner.cs

```diff
@@ -114,11 +113,11 @@ private void Awake()
         }
 
         /// <summary>
-        /// Starts the co-routine <see cref="SpawnPlayerCoroutine"/>.
+        /// Starts the co-routine <see cref="SpawnPlayer"/>.
```

Comment:
I think you forgot to update the comment. It seems that no coroutine is being started anymore.

Reply:
Oh, you're right. Fixed it now.